src/Application/UI/Presenter.php

```diff
@@ -504,7 +504,7 @@ public function formatLayoutTemplateFiles()
 		$name = $this->getName();
 		$presenter = substr($name, strrpos(':' . $name, ':'));
 		$layout = $this->layout ? $this->layout : 'layout';
-		$dir = dirname($this->getReflection()->getFileName());
+		$dir = dirname((new \ReflectionClass($this))->getFileName());
```

Comment:
I think that ReflectionObject::getProperties returns real object properties and ReflectionClass::getProperties returns default properties from a class.